style3.css

```diff
@@ -62,28 +65,17 @@ ul li ul{
 	padding: 0;
 	position: absolute;
 	top: 58px;
-	left: 1043px;
```

Comment:
27/09/2021: So I already re adjust the margin in the link above so I not need to reposition the drop down menu column to match with it anyone so I remove it This drop down menu column is for the query. Also this is the main reason that cause the error in the drop down menu